recipes/pds4_tools/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pds4_tools" %}
+{% set version = "1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pds4_tools-{{ version }}.zip
+  sha256: 8e6fdd0a9c4063c9a337b6ed3c52686da4f96bfba2a30e42ec57a6cf25fff459
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=2.6
+  run:
+    - numpy
+    - python >=2.6
+    - tk
+    - matplotlib-base
```

Comment:
Is matplotlib optional or not? If there's a good use case for not generating GUI elements then you may not want to require all users to install them.

The [setup.py](https://github.com/Small-Bodies-Node/pds4_tools/blob/50df4401897131fcccd37f5758d31a361ff34b37/setup.py#L90) lists matplotlib and tk/tkinter as optional.

Reply:
For users not interested in the GUI aspect it's optional. conda doesn't support marking optional dependencies directly as far as I can tell, and some people recommend then just marking those as required given the broad ecosystem support. Personally I am unsure which abroad is best.

Reply:
Thinking on this more, since other tools have used `pds4_tools` as a library for its read functionality, it's probably best to keep the optional dependencies for visualization as optional. 

However tk is not optional despite upstream claims. That's an oversight I need to fix upstream, it should be optional. Never caught it since tk generally ships with python, only incidentally caught it here due to the need for libx11. Thus I've removed matplotlib-base but am keeping tk for now.